### DISCUSSION TITLE
GitHub Sponsors?

### DISCUSSION THREAD

**Discussion by chrisandchris** [DigitalRuby/IPBan#165] [Q&A] [answered]
Hi

Do you plan on supporting GitHub Sponsors? I am willing to pay a one-time sponsoring as I like your software and it's great to have it available on a free plan, however I avoid using PayPal for any sort of payment.

Best regards
Chris

**Comment by jjxtra** [comment, marked as answer]
I've added this, it shows up in the side bar on the right and the readme now. Thanks for being willing to donate!
